vpr/src/base/vpr_context.h

```diff
@@ -608,32 +591,32 @@ class ServerContext : public Context {
      * Once calculated upon request, it provides the value for a specific critical path
      * to be rendered upon user request.
      */
-    std::vector<tatum::TimingPath> crit_paths_;
+    std::vector<tatum::TimingPath> crit_paths;
 
     /**
      * @brief Stores the selected critical path elements.
      *
      * This value is used to render the selected critical path elements upon client request.
      * The std::map key plays role of path index, where the element indexes are stored as std::set.
      */
-    std::map<std::size_t, std::set<std::size_t>> crit_path_element_indexes_;
+    std::map<std::size_t, std::set<std::size_t>> crit_path_element_indexes;
 
     /**
      * @brief Stores the flag indicating whether to draw the critical path contour.
      *
      * If the flag is set to true, the non-selected critical path elements will be drawn as a contour, while selected elements will be drawn as usual.
```

Comment:
what is a contour in this context?  An outline?  A flyline?  Clarifying what contour means here would be useful.

Reply:
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/f4d6824be043f7e0accbe2c2a26e71ff5c795423
(will be updated later)